apps/parse/lib/parse/vehicle_positions_json.ex

```diff
@@ -47,6 +48,14 @@ defmodule Parse.VehiclePositionsJson do
     []
   end
 
+  defp parse_consist([_ | _] = consist) do
+    consist
+    |> Enum.map(fn %{"label" => car_label} -> car_label end)
+    |> MapSet.new()
```

Comment:
For using this data in the future, we probably want to keep it as a list: the order of the labels is the order of the cars.